src/partials/header.html

```diff
@@ -31,11 +31,12 @@
       <div class="md-flex__cell md-flex__cell--shrink">
         <a href="{{ config.site_url | default(nav.homepage.url, true) | url }}"
             title="{{ config.site_name }}"
+            aria-label="{{ config.site_name }}"
```

Comment:
Adding an ARIA label + the alt tag on the image - how is this handled by screen readers?

Reply:
The Windows Narrator just says "sitename" once.

Reply:
And lighthouse complained about there missing a label to the a because the img inside of it was not enough, for whatever reason.

Reply:
Okay great, so this is ready to merge?

Reply:
Yes from my side!